components/NavBar/index.js

```diff
@@ -1,34 +1,70 @@
 import styled from "styled-components";
 import Link from "next/link";
+import { useRouter } from "next/router";
 
 export default function NavBar() {
+  const router = useRouter();
+
   return (
     <StyledNavigation>
-      <StyledLink href="/">
-        <img src="/Images/icons8-startseite-50.png" />
-      </StyledLink>
-      <StyledLink href="/WatchList">
-        <img src="/Images/icons8-lesezeichen-50.png" />
-      </StyledLink>
+      <StyledLink1 active={router.asPath === "/"} href="/" className="active">
+        HOME
+      </StyledLink1>
+      <StyledLink2 active={router.asPath === "/WatchList"} href="/WatchList">
+        LIST
+      </StyledLink2>
     </StyledNavigation>
   );
 }
 
 const StyledNavigation = styled.nav`
   display: flex;
   justify-content: space-between;
-  padding: 0rem 1rem;
+  height: 4rem;
+  padding: 0rem 1rem 0rem 1rem;
```

Comment:
You can use the shorthand here :)
```suggestion
  padding: 0 1rem;
```